index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
Hi Jorge, I suggest thta you can change this for something like that: Game over, refresh to play again 🧑‍💻 🤖!